README.md

```diff
@@ -2,11 +2,11 @@
 ==========
 
 [![Build Status](https://github.com/kotest/kotest/workflows/build/badge.svg)](https://github.com/kotest/kotest/actions)
-[<img src="https://img.shields.io/maven-central/v/io.kotest/kotest-framework-api-jvm.svg?label=latest%20release"/>](http://search.maven.org/#search|ga|1|kotest)
+[<img src="https://img.shields.io/maven-central/v/io.kotest/kotest-framework-api-jvm.svg?label=latest%20release"/>](https://search.maven.org/search?q=g:io.kotest%20OR%20g:io.kotest.extensions)
```

Comment:
Note: I have chosen a link here which covers the groups `io.kotest` and `io.kotest.extensions`.
However, since the extensions use a different versioning scheme, should this only cover `io.kotest`?

Reply:
I think that's fine.